sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md

```diff
@@ -388,6 +411,47 @@ make_buffer(
 Construct a SYCL buffer instance from a pointer to a Level Zero memory allocation. Please refer to <code>make_buffer</code>
 description above for semantics and restrictions.
 The additional <code>AvailableEvent</code> argument must be a valid SYCL event. The instance of the SYCL buffer class template being constructed must wait for the SYCL event parameter to signal that the memory native handle is ready to be used.
+</tr>
+
+<tr>
+<td>
+
+``` C++
+make_image(
+    const backend_input_t<backend::ext_oneapi_level_zero,
+                          image<Dimensions, AllocatorT>> &,
+    const context &Context, event AvailableEvent = {})
+```
+</td>
+<td>This API is available starting with revision 4 of this specification.
+
+Construct a SYCL image instance from a ze_image_handle_t. 
+
+Because LevelZero has no way of getting image information from an image, it must be provided.  The  <code>backend_input_t</code> is a struct type like so:
+``` C++
+struct type {
+    ze_image_handle_t ZeImageHandle;
+    sycl::image_channel_order ChanOrder;
+    sycl::image_channel_type ChanType;
+    sycl::range<Dimensions> Range;

```

Comment:
The SYCL specification [section 4.5.1.3](https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:backend-interoperability-make) defines the prototypes for `make_buffer`, and there are two overloads for that API (one with `event` and one without).  That section also specifies the prototypes for `maked_unsampled_image` and `make_sampled_image`, which also have two overloads each.

It is true that `make_image` is not in the SYCL 2020 spec because `image` is a 1.2.1 concept.  However, I think it makes sense for `make_image` to follow the pattern we have established in the spec.

It's also more efficient to have two overloads.  With a single overload, the application needs to construct an `event` whenever it calls `make_image`, even if it doesn't pass one.  I presume this is not a large amount of overhead, but it seems wasteful to construct an `event` object if it isn't needed.